### PR TITLE
Add support for soft position controller

### PR DESCRIPTION
**Pull request by brentyi**
As per berkeleyopenarms/blue_core#82, position control is being offloaded to our motor drivers.

Some specialized internal applications will still want support for our "legacy" position controller -- just added support for that.

Also did some more formatting tweaks :)

### CODE REVIEW

**Comment by wuphilipp** [comment]
Looks great! Will test and then merge!

File: blue_interface/blue_interface.py [hunk 1]
```diff
@@ -236,12 +262,19 @@ def get_gripper_effort(self):
         """
         return self._gripper_effort
 
-    def set_joint_positions(self, joint_positions, duration=0.0):
+    def set_joint_positions(self, joint_positions,
+                            duration=0.0, soft_position_control=False):
```
Comment (wuphilipp) [inline blue_interface/blue_interface.py]: This currently does not do anything?

Comment (brentyi) [reply blue_interface/blue_interface.py]: Spooky, maybe I forgot to push something -- will check tonight

**Comment by brentyi** [comment]
@wuphilipp fixed!

**Comment by wuphilipp** [comment]
Tested and works great!

**Pull request merged**